content/en/docs/gke/deploy/management-setup.md

```diff
@@ -9,118 +9,220 @@ This guide describes how to setup a management cluster which you will use to dep
 The management cluster is used to run [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview). Cloud Config Connector is a Kubernetes addon that allows you to manage Google Cloud resources through Kubernetes.
 
 While the management cluster can be deployed in the same project as your Kubeflow cluster, typically you will want to deploy
-it in a separate project used for administering one or more Kubeflow instances.
+it in a separate project used for administering one or more Kubeflow instances, because it will run with escalated permissions to create Google Cloud resources in the managed projects.
 
-Optionally, the cluster can be configured with [Anthos Config Management](https://cloud.google.com/anthos-config-management/docs) 
+Optionally, the cluster can be configured with [Anthos Config Management](https://cloud.google.com/anthos-config-management/docs)
 to manage Google Cloud infrastructure using GitOps.
 
 ## FAQs
 
-* Where is `kfctl`?
+- Where is `kfctl`?
 
-   `kfctl` is no longer being used to apply resources for Google Cloud, because required functionalities are now supported by generic tools including [Make](https://www.gnu.org/software/make/), [Kustomize](https://kustomize.io), [kpt](https://googlecontainertools.github.io/kpt/), and [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview).
+  `kfctl` is no longer being used to apply resources for Google Cloud, because required functionalities are now supported by generic tools including [Make](https://www.gnu.org/software/make/), [Kustomize](https://kustomize.io), [kpt](https://googlecontainertools.github.io/kpt/), and [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview).
 
-* Why do we use an extra management cluster to manage Google Cloud resources?
+- Why do we use an extra management cluster to manage Google Cloud resources?
 
-   The management cluster is very lightweight cluster that runs [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview). Cloud Config Connector makes it easier to configure Google Cloud resources using YAML and Kustomize.
+  The management cluster is very lightweight cluster that runs [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview). Cloud Config Connector makes it easier to configure Google Cloud resources using YAML and Kustomize.
 
-For a more detailed explanation of the changes affecting Kubeflow 1.1 on Google Cloud, read [kubeflow/gcp-blueprints #123](https://github.com/kubeflow/gcp-blueprints/issues/123).
+For a more detailed explanation of the drastic changes happened in vKubeflow 1.1 on Google Cloud, read [kubeflow/gcp-blueprints #123](https://github.com/kubeflow/gcp-blueprints/issues/123).
 
 ## Install the required tools
 
-1. Install gcloud components
+1.  Install gcloud components
```

Comment:
nit: remove extra whitespace? (and below)

Reply:
this was my auto formatting tool, it added the extra whitespace, so that this line will have the same indent as the next line (which indents by 4 spaces).
I think we should standardize on a tool that is compatible with the format this website accepts, that could save a lot of formatting efforts.